docs/Guide_To_The_Linux_Installers.md

```diff
@@ -0,0 +1,599 @@
+    + Current Options :
+      + Alpine
+      + Debian
+      + RHEL
+      + Suse
+      + all (This will attempt to run for all of the available distributions)
+
+There are a number of restrictions around combinations of architecture and platforms, these are detailed in Appendix A, below, but the [jenkinsfile](https://github.com/adoptium/installer/blob/master/linux/Jenkinsfile) has been coded to exclude these from the build processes, details here are included for reference.
+
+  ### 1.2. Gradle / Docker Package Build & Test Process
+
+The jenkins job above once triggered, performs a build, using gradle, and a dockerfile to produce, installer packages for the supported versions of each of the 4 distributions that we currently support, ie .apk (Alpine), .deb (Debian) & .rpm (RHEL & Suse). The versions of each specific distribution supported can be viewed in the [jenkinsfile](https://github.com/adoptium/installer/blob/master/linux/Jenkinsfile), by searching for the deb_versions variable, or the distro_Package variable, the table below shows the current supported versions, but this does change on a regular basis.
+
+Supported Linux Distros:
+Distribution Type| Supported Versions
+----------|---------
+Apk (Alpine)| All supported version.
+Deb (Debian)| Trixie (Debian 13)<br>Bookworm (Debian 12)</br>Bullseye (Debian 11)<br>Buster (Debian 10)</br> Noble (Ubuntu 24.04)</br>Jammy (Ubuntu 24.04)</br>Focal (Ubuntu 20.04)</br>Bionic (Ubuntu 18.04)
+RPM (RHEL)| centos 7</br> rocky 8</br>RHEL7 , RHEL8 & RHEL9</br> Fedora 35, 36, 37, 38 ,39 , 40</br>Oracle Linux 7 & 8</br>Amazon Linux 2
+RPM(SUSE) | Opensuse 15.3</br>Opensuse 15.4</br>Opensuse 15.5</br>SLES 12</br>SLES15
+
+As part of this process, the packages are first built, using the appropriate mechanisms, and then a set of unit tests are executed to ensure the builds have completed correctly.
+
+  ### 1.3. Package Signing & Upload
+
+  The final part of the jenkins job is triggered when either/both of the enable enableGpgSigning and uploadPackage options are selected. 
+
+  The Gpg signing uses the Adoptium Gpg key to sign the installer packages prior to their upload to the [Jfrog artifactory, hosted at packages.adoptium.net](https://packages.adoptium.net/)
+
+
+
+## 2. Preparing The Source Code For Package Builds & Releases
+
+Prior to building and releasing any packages, the source code that controls the process requires a number of updates. These updates typically happen once all of the linux platforms for a particular Java version have been released and the binaries uploaded to Github.
+
+Prior to updating the source code, its important to have an understanding of how the source code is structured so that the correct files can be updated. The callout below, highlights the structure of the source code so for example underneath the linux top level directory, a selection of either jdk/jre, followed by the distribution, and then the vendor (temurin in the example), followed by the Java version. 

```

Comment:
```suggestion
Prior to updating the source code, its important to have an understanding of how the source code is structured so that the correct files can be updated. The callout below, highlights the structure of the source code, for example underneath the `linux` top level directory, a selection of either jdk/jre, followed by the distribution, and then the vendor (temurin in the example), followed by the Java version. 

```